src/main/java/com/rultor/agents/github/qtn/QnRelease.java

```diff
@@ -65,31 +66,56 @@
     @Override
     public Req understand(final Comment.Smart comment,
         final URI home) throws IOException {
-        new Answer(comment).post(
-            true,
-            String.format(
-                QnRelease.PHRASES.getString("QnRelease.start"),
-                home.toASCIIString()
-            )
-        );
         final Issue issue = comment.issue();
         Logger.info(
             this, "release request found in %s#%d, comment #%d",
             issue.repo().coordinates(), issue.number(), comment.number()
         );
-        return new Req.Simple(
-            "release",
-            new ImmutableMap.Builder<String, String>()
-                .put("head_branch", "master")
-                .put(
-                    "head",
+        final Req req;
+        final Matcher matcher = Pattern.compile(".*release.*`(.+)`.*")
+            .matcher(comment.body());
+        if (matcher.matches()) {
```

Comment:
@original-brownbear instead of `matches()` we could use `find()` and avoid `.*` on both sides